docs/ops.html

```diff
@@ -4027,8 +4027,8 @@ <h5 class="anchor-heading"><a id="static_versus_dynamic_kraft_quorums" class="an
   When using a static quorum, the configuration file for each broker and controller must specify
   the IDs, hostnames, and ports of all controllers in <code>controller.quorum.voters</code>.<p>
 
-  In contrast, when using a dynamic quorum, you should set
-  <code>controller.quorum.bootstrap.servers</code> instead. This configuration key need not
+  In contrast, when using a dynamic quorum, <code>controller.quorum.bootstrap.servers</code>
+  is set instead and <code>controller.quorum.voters</code> must not be set. This configuration key need not
```

Comment:
btw, can you improve the config description as well? something like

[controller.quorum.voters](https://kafka.apache.org/documentation/#brokerconfigs_controller.quorum.voters)
Map of id/endpoint information for the set of static voters in a comma-separated list of {id}@{host}:{port} entries. This is the old way of defining membership for controller quorums and should NOT be set if using dynamic quorums. See [Static versus Dynamic KRaft Quorums](https://kafka.apache.org/documentation/#static_versus_dynamic_kraft_quorums) for details.

For example: 1@localhost:9092,2@localhost:9093,3@localhost:9094